code/src/components/Icons.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+export const Icons = () => {
+  return (
+    <div class="icons">
+      <img className="heart icon" src="/icons/heart.svg" alt="heart-icon"/>
+      <img className="play icon" src="/icons/play.svg" alt="play-icon"/>
+      <img className="dots icon" src="/icons/dots.svg" alt="dots-icon"/>
+    </div>
+  );
```

Comment:
We did the same! Although your icons does not show in Safari (mobile) when we tried it. Do you know why?
/ Ida & Emma